### PR TITLE
chore: Upgrade to electron 7.x

### PR DESCRIPTION
**Pull request by deepak1556**
**Chromium:** 78.0.3904.130
**Nodejs:** 12.8.1
**Electron:** 7.1.7

### CODE REVIEW

**Review by bpasero — APPROVED** [review]
Changes LGTM but I suggest this **timeline**:
* consistent green CI builds on all platforms
* green exploration build for selfhosting during this milestone
* full endgame testing on all platforms on exploration build (including dev setup)
* push to insiders next milestone if no blockers

File: src/vs/platform/electron/electron-main/electronMainService.ts [hunk 1]
```diff
@@ -366,15 +366,13 @@ export class ElectronMainService implements IElectronMainService {
 	//#region Connectivity
 
 	async resolveProxy(windowId: number | undefined, url: string): Promise<string | undefined> {
```
Comment (ZuBB) [inline src/vs/platform/electron/electron-main/electronMainService.ts]: (maybe I am missing smth, but still..) Is current version of return type still valid after change is is below?

**Comment by webczat** [comment]
What prevents this to be merged? It is required for linux accessibility so I am waiting impatiently...

File: extensions/vscode-api-tests/src/singlefolder-tests/webview.test.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ const webviewId = 'myWebview';
 
 const testDocument = join(vscode.workspace.rootPath || '', './bower.json');
 
-suite('Webview tests', () => {
+suite.skip('Webview tests', () => {
```
Comment (mjbvz) [inline extensions/vscode-api-tests/src/singlefolder-tests/webview.test.ts]: Hey looks like web views are broken under electron due to `Prevented webview attach`. I'm looking into it but wonder if you have any idea what that may be?

Comment (mjbvz) [reply extensions/vscode-api-tests/src/singlefolder-tests/webview.test.ts]: Pushed fix with c8123dee962847abcbd988d8c180ae1dec0b56d5 and re-enabled the tests

**Pull request merged**